lib/v8.js

```diff
@@ -15,5 +15,19 @@
 'use strict';
 
 var v8binding = process.binding('v8');
-exports.getHeapStatistics = v8binding.getHeapStatistics;
+
+exports.getHeapStatistics = function() {
+  var data = {};
```

Comment:
@vkurchatkin I'm sorry. I misread the initial implementation and interpreted the `maxresident` to suggest there was a memory leak. I thought I saw `var data = {};` outside of the `getHeapStatistics()` function. Which would have meant every call to `getHeapStatistics()` would have attached memory to the same object, leaving the previously allocated memory hanging around and unable to be collected. My mistake. :)